RELEASE_NOTES.md

```diff
@@ -1,5 +1,6 @@
 # Release Notes
-
+## 3.3.0
+*   Added option to resize images to meet expected dimensions before comparison using compareMisSizedImages() 
```

Comment:
outdated description for this update. 
Please, provide up-to-date description based on your last changes.

Reply:
Thanks, @SmashSide for your catch.